testplan/web_ui/testing/src/AssertionPane/AssertionTypes/basicAssertionUtils.js

```diff
@@ -127,13 +127,18 @@ function prepareEqualContent(assertion, defaultContent) {
  * @private
  */
 function prepareNotEqualContent(assertion, defaultContent) {
-  const leftContent = <span>
-    &lt;not&gt; {_.isNil(assertion.second) ? "" : String(assertion.second)}
-  </span>;
+  let leftContent;
+  let rightContent;
+
+  leftContent = _.isNil(assertion.second) ? "None" : String(assertion.second);
+  rightContent = _.isNil(assertion.first) ? "None" : String(assertion.first);
+
+  const prefix = <span>&lt;not&gt; </span>
```

Comment:
This separation is useful in case the prefix will see a different styling (say colored red) or formatting (added as cell prefix rather than inline).